packages/new-polymath-shared/src/formatters/index.ts

```diff
@@ -76,10 +76,7 @@ export const toTokens = (
     num = new BigNumber(`${value}`);
   }
 
-  return num
-    .precision(18)
-    .decimalPlaces(decimals)
-    .toFormat();
+  return num.precision(18).toFixed(decimals);
```

Comment:
That's a tricky one, here you loose the capabilities of the `toFormat()` function which was transforming things like `500000` to `500,000`.

Reply:
Agreed. This was oversight on my part.